docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Modules** subtab. All modules you have added to your robot appear under the **Deployed** section.
+
+This pane lists the models provided by the module, any [components](/components/) on your robot that are currently using those models, and allows you to configure [how the module updates](#configure-version-update-management-for-a-registry-module) when a new version is available from the Viam Registry.
+
+{{<imgproc src="extend/modular-resources/configure/conf-module-from-registry.png" resize="1000x" declaredimensions=true alt="The module subtab of the config tab showing the realsense custom module configuration pane includes the update management section showing version update management options version type, set to Patch (X.Y.Z) and version set to 0.0.3">}}
+
+#### Configure version update management for a Registry module
+
+When you add and configure a module on your robot, the module is automatically set to pin to the specific patch release version you added.
+
+Depending on how you use the module, and what role your robot plays, you may wish to control how that deployed module updates when a new version of the module is made available in the Viam Registry.
+Updating to a newer version of a module brings new functionality and bug fixes, but requires restarting the module to apply the update.
+
+You can use the **Version type** menu of a module's configuration pane under the **Modules** subtab to configure version update behavior for your module:
 
+- **Patch (X.Y.Z)**: Do not update to any other version. This is the default.
+- **Minor (X.Y.*)**: Only update to newer patch releases of the same minor release branch.
+  The module will automatically restart and update itself whenever new updates within the same minor release are available in the Viam Registry.
+  For example, use this option to permit a module with version `1.2.3` to update to version `1.2.4` or `1.2.5` but not `1.3.0` or `2.0.0`.
+- **Major (X.*)**: Only update to newer minor releases of the same major release branch.
+  The module will automatically restart and update itself whenever new updates within the same major release are available in the Viam Registry.
+  For example, use this option to permit a module with version `1.2.3` to update to version `1.2.4` or `1.3.0` but not `2.0.0` or `3.0.0`.
+- **Latest** Always update to the latest version of this module available from the Viam Registry as soon as a new version becomes available.
+
+When using the **Patch (X.Y.Z)** version type, you may select any patch version of the module from the **Version** drop down menu, including past versions if desired.
+
+The current deployed version of your module and the latest version of that module available from the Viam Registry are shown on this pane for your reference.
+
+{{% alert title="Updating production robots" color="caution" %}}
```

Comment:
```suggestion
{{% alert title="Caution" color="caution" %}}
```
I *think* these are kinda supposed to be uniform--but will add to tomorrow's meeting agenda because I've been wondering